contrib/experiments/test_messaging.py

```diff
@@ -202,9 +199,9 @@ def on_receive_pdu(self, pdu):
             )
 
     # def on_state_change(self, pdu):
```

Comment:
Looks like this code should just go away (and the second below it)?

Reply:
Good point.